web/src/Search.css

```diff
@@ -0,0 +1,55 @@
+html {
+    font-size: 12px;
+}
+  
+body {
+  background-color: #F2F2F2;
+}
+  
+.Search {
+  padding: 10px;
+  background-image: url("https://i.imgur.com/MPcDf2v.jpg");
+  background-repeat: repeat;
+  box-shadow: 0px 1px 5px 1px gray;
+  }
+
+.SearchWrapped {
+  padding: 100px 0px 50px 100px;
+}
+  
+.SearchTitle {
+  font-size: 2.5rem;
+  color: #fff;
+  width: 480px;
+}
+  
+.SearchHint {
+  padding: 5px 0px 5px 0px;
+  color: #fff;
+  width: 500px;
+}
+  
+.input[type=text] {
```

Comment:
@elkmare can you change this to just class? e.g. `.inputText`, and then in the component `className={styles.inputText}`

Reply:
Done 😄